suite-native/forms/package.json

```diff
@@ -11,7 +11,7 @@
         "type-check": "tsc --build"
     },
     "dependencies": {
-        "@hookform/resolvers": "3.1.0",
+        "@hookform/resolvers": "3.3.2",
```

Comment:
Is this safe to do for `suite-native`? Who shall I ask to approve this from the Mobile Team?

Reply:
you can tag whole @trezor/suite-mobile team or ask @Nodonisko

Reply:
Should be safe, it's just minor version and we use hook form in super basic form.